lib/active_setting/loader.rb

```diff
@@ -6,14 +6,27 @@ def self.load_settings(filename = self.config_filename)
       settings_config(filename).each do |category_name, settings|
         settings.each do |setting_name, values|
           Setting.register(setting_name.to_sym, values.merge(
-            :data_type => values['type'],
-            :category => category_name,
-            :name => setting_name
+            data_type: values['type'],
```

Comment:
Indent the first parameter one step more than `values.merge(`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.